src/model/chat.ts

```diff
@@ -214,22 +216,22 @@ function logResponse(args: {
       completion_tokens: number;
       prompt_tokens: number;
     };
-    latency?: number;
     cached: boolean;
+    latency?: number;
```

Comment:
is this still optional with the changes above?

Reply:
I'm not sure; the app-level fields vs intrinsic response fields are a bit of a mess atm